src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -212,17 +219,22 @@ public static AttributeCollection FromExisting(AttributeCollection existing, par
         /// <summary>
         /// Determines if this collection of attributes has the specified attribute.
         /// </summary>
-        public bool Contains(Attribute attribute)
+        public bool Contains(Attribute? attribute)
```

Comment:
Why did you change this to allow null?  I'd prefer to just leave this method the way it was.